flocker/ca/_ca.py

```diff
@@ -4,6 +4,25 @@
 Low-level logic for a certificate authority.
 
 Uses RSA 4096-bit + SHA 256.
+
+We have three categories of certificates:
+
+1. Control service. Needs to be validated over HTTP by 3rd party clients.
+2. Node agents. Needs to encode a node UUID.
+3. API clients. Used by HTTP API clients to authenticate.
```

Comment:
Each item in this list comes with an explanation of a different area.  It would be nice to see information for all three areas for all three items.